### PR TITLE
Auto subs

### PR DESCRIPTION
**Pull request by yurak**

### CODE REVIEW

File: config/database.yml [hunk 1]
```diff
@@ -11,7 +11,7 @@ default: &default
 
 development:
   <<: *default
-  database: db/development.sqlite3
+  database: db/production.sqlite3
```
Comment (yurak) [inline config/database.yml]: need to fix

File: app/views/tournament_rounds/show.html.haml [hunk 2]
```diff
@@ -31,6 +32,12 @@
           - if tournament_round.tours.locked.any?
             .round-action-link
               = link_to  t('round.postpone'), tours_update_tournament_round_path(tournament_round, status: :postponed), method: :put, data: { confirm: t('header.are_you_sure') }
-          - if tournament_round.finished? && tournament_round.tours.locked_postponed.any?
+          - if true
```
Comment (BohdanChaban) [inline app/views/tournament_rounds/show.html.haml]: return statement

File: config/database.yml [hunk 3]
```diff
@@ -11,7 +11,7 @@ default: &default
 
 development:
   <<: *default
-  database: db/development.sqlite3
+  database: db/staging_fresh.sqlite3
```
Comment (BohdanChaban) [inline config/database.yml]: development

File: app/controllers/tournament_rounds_controller.rb [hunk 4]
```diff
@@ -57,6 +71,6 @@ def update_params
   end
 
   def tournament_round
-    @tournament_round ||= TournamentRound.find(params[:id])
+    @tournament_round ||= TournamentRound.find(params[:id] || params['tournament_round_id'])
```
Comment (BohdanChaban) [inline app/controllers/tournament_rounds_controller.rb]: prefer to use symbol keys

File: app/models/league.rb [hunk 5]
```diff
@@ -31,6 +31,10 @@ class League < ApplicationRecord
       .order(Arel.sql('CASE WHEN division_id IS NULL THEN 1 ELSE 0 END, division_id ASC'))
   }
 
+  def divisison_with_name
```
Comment (BohdanChaban) [inline app/models/league.rb]: a typo in the word `division`

File: app/services/substitutes/auto_bot.rb [hunk 6]
```diff
@@ -0,0 +1,130 @@
...
+    end
+  end
+end
+
+# lineup = Match.find(  ).host_lineup
+
+# Substitutes::AutoBot.new(lineup, preview: true).process
+
+# lineup = Lineup.find(42624)
+# Substitutes::AutoBot.new(lineup, preview: true).process
```
Comment (BohdanChaban) [inline app/services/substitutes/auto_bot.rb]: remove comments

**Pull request merged**